sql/migrations/2023-05-15-work-types/up.sql

```diff
@@ -0,0 +1,179 @@
+-- Add new columns for hierarchy, description and deprecation
+
+BEGIN TRANSACTION;
+
+ALTER TABLE bookbrainz.work_type ADD COLUMN description TEXT;
+ALTER TABLE bookbrainz.work_type ADD COLUMN parent_id INT;
+ALTER TABLE bookbrainz.work_type ADD COLUMN child_order INT NOT NULL DEFAULT 0;
+ALTER TABLE bookbrainz.work_type ADD COLUMN deprecated BOOLEAN NOT NULL DEFAULT FALSE;
+
+ALTER TABLE bookbrainz.work_type ADD FOREIGN KEY (parent_id) REFERENCES bookbrainz.work_type (id);
+
+COMMIT;
+
+
+-- Add new types and modify existing ones — see https://tickets.metabrainz.org/browse/BB-740
+
+-- Top-level types first
+BEGIN TRANSACTION;
+
+	-- Non-fiction, id 8
+	UPDATE bookbrainz.work_type
+		SET description='Prose work that is not fiction.',
+			parent_id=NULL,
+			child_order=1
+		WHERE id=8;
+
+	-- Poem, id 4
+	UPDATE bookbrainz.work_type
+		SET description='A work of poetry; a non-prosaic composition that uses stylistic and rhythmic qualities of language to evoke meanings in addition to, or in place of, ostensible meaning. Poetry is very variable and particularly difficult to define; generally any work described as poetry should be considered of the poem work type.',
+			parent_id=NULL,
+			child_order=2
+		WHERE id=4;
+
+	INSERT INTO bookbrainz.work_type ("id","label","description","parent_id","child_order")
+	VALUES
+		(13, 'Fiction','Literary work portraying individuals or events that are imaginary, though it may be based on a true story or situation.',NULL,0);
```

Comment:
```suggestion
		(13,'Fiction','Literary work portraying individuals or events that are imaginary, though it may be based on a true story or situation.',NULL,0);
```
All other commas are not padded with spaces ;)